core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -420,6 +424,12 @@ public static ConfigDef baseConfigDef() {
             ENABLE_GZIP_COMPRESSION_DEFAULT,
             Importance.LOW,
             ENABLE_GZIP_COMPRESSION_DOC
+        ).define(
+            WEBSOCKET_PATH_PREFIX_CONFIG,
+            Type.STRING,
+            "/ws",
+            Importance.LOW,
```

Comment:
I think this should not even be documented. this configuration is only meant to be set programmatically. Is there a way to enforce that @ewencp ?

Reply:
AK ConfigDef has a defineInternal that should not get documented but I think that happened way after we split off from it, so probably not without some other changes in common.